modules/authorization/proc-sending-request-to-the-rbac-rest-api-by-using-curl.adoc

```diff
@@ -0,0 +1,113 @@
+[id='proc-sending-requests-to-the-rbac-rest-api-by-using-curl_{context}']
+= Sending requests to the RBAC REST API by using the curl utility
+
+You can send RBAC REST API requests by using the curl utility.
+
+.Prerequisites
+* xref:enabling-and-giving-access-to-rbac[You have access to the RBAC feature].
+
+.Procedure
+include::snip-finding-bearer-token.adoc[]
+
+. In a terminal, run the curl command and review the response:
++
+--
+.Default request
+[source,subs="+attributes,+quotes"]
+----
+curl -v -H "Content-Type: application/json" \
+  -H "Authorization: Bearer _<token>_" \
+  -X __<method>__ "{my-product-url}/__<endpoint>__" \
+----
+
+.POST request requiring JSON body data
+[source,subs="+attributes,+quotes"]
+----
+curl -v -H "Content-Type: application/json" \
+  -H "Authorization: Bearer _<token>_" \
+  -X POST "{my-product-url}/__<endpoint>__" \
+  -d __<body>__
+----
+
+__<token>__::
+Enter your saved authorization token.
+
+__<method>__::
+Enter the HTTP method for your xref:ref-rbac-rest-api-endpoints_{context}[API endpoint].
+
+* `GET`: To retrieve specified information from a specified resource endpoint.
+* `POST`: To create or update a resource.
+* `PUT`: To update a resource.
+* `DELETE`: To delete a resource.
+
+pass:c,a,q[{my-product-url}]::
+Enter your {product-short} URL.
+
+__<endpoint>__::
+Enter the xref:ref-rbac-rest-api-endpoints_{context}[API endpoint] to which you want to send a request, such as `/api/permission/policies`.
+
+__<body>__::
+Enter the JSON body with data that your xref:ref-rbac-rest-api-endpoints_{context}[API endpoint] might need with the HTTP `POST` request.
+
+.Example request to create or update a role
+[source,subs="+attributes,+quotes"]
+----
```

Comment:
With "create" I agree. About "update", I think we have another response body format and url pattern too:

```
curl -v -H "Content-Type: application/json" \
  -H "Authorization: Bearer _<token>_" \
  -X PUT "{my-product-url}/api/permission/roles/role/default/test" \
  -d '{
          "oldRole": {
            "memberReferences":  [ "group:default/example" ],
            "name": "role:default/test",
          },
          "newRole": {
            "memberReferences": [ "group:default/example", "user:default/test" ],
            "name": "role:default/test"
          }
        }'
```